packages/arp/arp.0.1.1/opam

```diff
@@ -29,6 +29,11 @@ depopts: [
   "tcpip"
   "lwt"
 ]
+
+conflicts: [
+  "tcpip" {< "3.0.0"}
+]
```

Comment:
that's wrong.. `arp < 0.2.0` should conflict with `tcpip  > 3.0.0`, not `<`... only `arp=0.2.0` got support for `tcpip-3.0.0`